controllers/home_controller.rb

```diff
@@ -30,10 +34,6 @@ class HomeController < ApplicationController
       end
     end
 
-    get "doc/api" do
```

Comment:
why removing this?

Reply:
this was the old route that redirects to `/documentation`. now we are using the route /mod-api/doc

Reply:
MOD API spec says it need to be `/mod-api/doc/api`

Reply:
Okk will change it